### DISCUSSION TITLE
Npm package

### DISCUSSION THREAD

**Discussion by galvakojis** [gildas-lormeau/zip.js#356] [General]
I am not sure how it works, but could you update this package from npm?
https://www.npmjs.com/package/zip-js

it took me some time to realize that I have an old version

**Comment by 0f-0b** [comment]
The latest version is available at <https://www.npmjs.com/package/@zip.js/zip.js>.

**Comment by gildas-lormeau** [comment]
Indeed, I'm not the person who published https://www.npmjs.com/package/zip-js which is the legacy version of zip.js, maybe @johanpoirier could remove it from NPM to fix this?
Edit: since this old version is still used, I'm unsure that would be a good idea to remove it actually. Maybe adding a note would in the README would suffice.

**Comment by galvakojis** [comment]
some note or warning would be enough, cause at the moment it can be confusing 
![image](https://user-images.githubusercontent.com/17315578/182099427-e78642bb-cde0-4900-ada0-548e8229099d.png)

**Comment by gildas-lormeau** [comment]
@johanpoirier It looks like you updated your README. However, you have to bump your version an publish it in order to update the README on NPM.

**Reply by johanpoirier** [reply]
With some huge delay, it is published (sorry I had a lot of health issues the past year).
I marked the package as deprecated.

**Reply by gildas-lormeau** [reply]
No problem @johanpoirier, thank you very much for the update. I hope your health is improving, that's the most important thing.
